### DISCUSSION TITLE
Air quality only Weather entities

### DISCUSSION THREAD

**Discussion by decompil3d** [home-assistant/architecture#879] [Entity Models]
Hi folks,

In review for home-assistant/core#88865, @frenck requested that we consider using a Weather entity to represent air quality forecast data. I'm opening this arch discussion to discuss the entity model change required for that.

## The scenario

The AirNow integration currently creates a set of numeric sensors to track current air quality across a few dimensions. The AirNow API also provides forecast data for AQI. I'd like to track that forecast data in Home Assistant.

## The entity model change

Weather entities already support the concept of a forecast, so they are a good fit for this sort of data. However, [Weather entities](https://developers.home-assistant.io/docs/core/entity/weather) currently require `condition` and `native_temperature` properties both for current state and for forecast items.

### Proposal

1. Adjust the Weather entity model to no longer require those properties
2. Add properties for `air_quality` (or similar name) both for current and forecast conditions

Then, we would adjust the AirNow integration to create a Weather entity that covers current and forecast AQI. I suspect that we'd remove the existing Sensor for AQI, but leave the other Sensors for folks who want to operate directly on PM2.5, PM10, etc. But that's outside the scope of this arch discussion.

**Comment by balloob** [comment]
I think that we should create a generic way to handle forecasts, for anything. I've played with the idea of [data sources](https://docs.google.com/document/d/1-UGjbGkwCO12e-krcO-TfnAOuvjMhwGRJYQlexcGSCw/edit#) in the past, but that got kinda stuck for being too similar to services.

**Comment by gjohansson-ST** [comment]
I think this would be a better fit with #667 instead of using the `WeatherEntity` as also partially suggested by @balloob
